### ISSUE TITLE
Could you guys update the docs with dependencies required?

### ISSUE THREAD

**Issue by nkiermaier** [phpenv/phpenv#71]
I had  to install autoconf, bison, libjpeg-dev, libpng12-dev, re2c and had to research these because the 
error messages weren't clear and required bashing my head into the wall one dependency at a time. 

One nice sudo apt-get list would be a huge timesaver for  people...

This will scare away less experienced people in my opinion. 

Also, could you guys make the default repo clone branch master?

**Comment by debo** [comment]
Sadly that won't happen because of the issue I linked to you in the other thread, but you can find a farily comprehensive list of needed dependencies here:

https://github.com/phpenv/phpenv/issues/6#issuecomment-13405869

I hope it helps

**Comment by nkiermaier** [comment]
Thanks, I noticed there's a fork of the project that I did not know about before.

**Comment by debo** [comment]
No problem, good that you found it now...

**Comment by nkiermaier** [comment]
Thanks much for responding.  Did anyone  ever  find  out what happened to nickl?

**Comment by debo** [comment]
Unfortunately not, I am still in discussion with Github support to see if at least Dominic, the creator of phpenv can be given access to the repo but so far it's proving difficult.

**Comment by nkiermaier** [comment]
Wow, sad.   As a compromise, could Github at least add a link to the project readme directing to the new fork? 

To direct us noobs to the correct place. 

Thanks. Hope your week is well.

Nick Kiermaier

**Comment by debo** [comment]
Nope, I asked that too and they are not in the position to alter the original repo.

**Comment by jpuck** [comment]
@debo what's the recommended fork you referenced? There are quite a few in the network. Thanks!

**Comment by debo** [comment]
@jpuck the one that I am currently using is [this one](https://github.com/madumlao/phpenv) which is the one that was recommended in [this discussion](https://github.com/phpenv/phpenv/issues/57) and was quite active at the time. Now it seems stale but it proven stable for me, in all fairness most of the work is done by php-build anyway so... if you know of any other alternative which is more up to date please do let me know. 

I hope it helps. Cheers

**Comment by madumlao** [comment]
I'll be working on this, using an updated version of [phpenv-common-deps-install](https://github.com/rogeriopradoj/phpenv-common-deps-install) as reference.
